hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/LogEventView.avdl

```diff
@@ -0,0 +1,15 @@
+@namespace("org.hypertrace.viewgenerator.api")
```

Comment:
We have to consider in the follow up PR for not setting logs here - https://github.com/hypertrace/data-model/blob/main/data-model/src/main/avro/eventfields/jaeger/JaegerFields.avdl

Reply:
sure